.github/workflows/checks.yml

```
@@ -29,6 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       FORCE_COLOR: 1
+      CI: true # needed for "eth-gas-reporter" to produce "gasReporterOutput.json"
```

Let's keep a explicit reference for this.

```suggestion
      # Needed for "eth-gas-reporter" to produce a "gasReporterOutput.json"
      # according to https://github.com/cgewecke/hardhat-gas-reporter/blob/ec80a50b14967290b290fc36192a14fa09f4d3c2/README.md?plain=1#L145
      CI: true
```

---

line 145 really is not what we want to point to.

---

Is that ok ?

```suggestion
      # Needed for "eth-gas-reporter" to produce a "gasReporterOutput.json"
      # according to https://github.com/cgewecke/eth-gas-reporter/blob/master/docs/gasReporterOutput.md
      CI: true
```

---

Yeah, you're right, I got it from clicking the `Copy Permalink` option. I'd like to reference the current commit so it doesn't get lost.

Let's do:

```suggestion
      # Needed for "eth-gas-reporter" to produce a "gasReporterOutput.json"
      # according to https://github.com/cgewecke/hardhat-gas-reporter/blob/ec80a50b14967290b290fc36192a14fa09f4d3c2/README.md
      CI: true
```